modules/ROOT/pages/showcase-apps.adoc

```diff
@@ -13,7 +13,7 @@ include::_partials/showcase-apps/index.adoc[leveloffset=1]
 
 
 // start showcase-apps/service-setup.adoc[leveloffset=1]
-include::_partials/showcase-apps/service-setup.adoc[leveloffset=2,tags=!excludeDownstream]
```

Comment:
@finp, I've tested the tagging system usptream, and it now works as expected :+1: